### ISSUE TITLE
go-audio command not found

### ISSUE THREAD

**Issue by MichelangeloAM** [Harry-027/go-audio#1]
Hi, amazing work, I'm eager to try it!

Only problem is that, after installing everything needed, cloning, using the `make download` and `make setup` commands and having no errors, I run the last command and get a "command not found" error.

I attached a screenshot of my terminal.

<img width="674" alt="Schermata 2020-11-19 alle 18 47 34" src="https://user-images.githubusercontent.com/48190785/99703692-b29d1780-2a97-11eb-84ab-c13f823579a1.png">

I really hope you can give me a solution, have a good day!

**Comment by Harry-027** [comment]
@Ziomike98, did you check if the go-audio binary file got installed in the go bin directory ? If yes, then your OS might not be aware of GOPATH. If you set GOPATH correctly or add the bin directory path to PATH env. variable, it should work.

**Comment by MichelangeloAM** [comment]
I tried setting it, I'm on macOS and after following some instructions I found it still doesn't work.

Any chance you can give me a step-by-step mini guide?

I tried following these [instructions ](https://gist.github.com/molivier/271bba5d67de1583a8e3), but they didn't work...

Many thanks and also thanks for the instant response!

**Comment by MichelangeloAM** [comment]
1) I added the Path;

<img width="561" alt="Schermata 2020-11-20 alle 16 11 14" src="https://user-images.githubusercontent.com/48190785/99815752-0583d700-2b4b-11eb-992b-153d53398cf2.png">

2) Placed the "go-audio" folder into /Users/michy/go/bin;

3) Sent this commands in the command prompt:
`cd /Users/michy/go/bin/go-audio`
`make download`
`make setup`

and I get this error now:
<img width="568" alt="Schermata 2020-11-20 alle 16 13 57" src="https://user-images.githubusercontent.com/48190785/99816005-66131400-2b4b-11eb-94e4-a932ca327004.png">

What did I do wrong? I understood the modifications you told me to do, but now it says that I already built it, correct?

I'm assuming the folder placement could be wrong, any chance it's that?
